### DISCUSSION TITLE
discutir sobre checklist de code review

### DISCUSSION THREAD

**Discussion by LuccasTraumer** [techexpertspro/portal-ascepa#20] [Ideas]
Visando termos uma "padronizacao" de o que observar e analisar nos code reviews.
Ex: Imports desnecessarios, variaveis inutilizaveis, templates desnecessarios, testar localmente, etc...

**Comment by arthuralveso** [comment]
Boa noite! Acredito que alguns dos elementos podem ser resolvidos diretamente com a instalação do husky ja configurando pra rodar os testes locais, verificar lint que já analisa a questão dos imports e variaveis não utilizados. Em tese ja subiria um commit mais "limpo"

**Reply by LuccasTraumer** [reply]
Bom dia, abri mais esse item para termos um padrão de o que visualizar em code review, por exemplo os imports devem ser primeiro do Angular, depois de libs externas e por últimos services e imports do Projeto. Se vamos validar lógica do que esta sendo desenvolvido ou nomenclatura e legibilidade.

Coisas que devemos ter ciência e que as ferramentas não consigam pegar. Não sei se consegui me fazer entender mas era isso kkkk

**Reply by andrewarosario** [reply]
É exatamente isso. Todos esses detalhes de lint e imports vamos deixar para as ferramentas fazerem esse papel. Até a organização dos imports o VSCode faz com a opção `autoOrganizeImports`.
A ideia é focar o code review 100% na implementação 😉
